docs/Inputs.md

```diff
@@ -197,19 +197,49 @@ Refer to [Material UI Datepicker documentation](http://www.material-ui.com/#/com
 When you want to display a record property in an `<Edit>` form without letting users update it (such as for auto-incremented primary keys), use the `<DisabledInput>`:
 
 ``` js
-import { DisabledInput } from 'admin-on-rest/mui';
+import { DisabledInput } from 'admin-on-rest/lib/mui';
 
 <DisabledInput source="id" />
 ```
 
 ![DisabledInput](./img/disabled-input.png)
 
+**Tip**: To add non-editable fields to the `<Edit>` view, you can also use one of admin-on-rest `Field` components:
+
+```js
+// in src/posts.js
+import { Edit, DisabledInput, LongTextInput, TextField, TextInput } from 'admin-on-rest/lib/mui';
```

Comment:
Remove `DisabledInput` and `TextInput` as they are not required in this example.